app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -305,10 +328,10 @@ public void onSuccess(Object var1) {
                         public final void onSuccess(Location location) {
                             if (location != null) {
                                 MapsActivity.this.lastLocation = location;
-                                LatLng currentLatLng = new LatLng(location.getLatitude(),
-                                        location.getLongitude());
+                                LatLng currentLatLng = new LatLng(42.4470171,
```

Comment:
Is there a reason we've moved our system to hardcoded latlongs instead of zooming in on the user's location?

Reply:
No, I was just using it to test something. I'll change it back.